dax-data-set-descriptors/jfk.yaml

```diff
@@ -72,3 +72,9 @@ related_assets:
     description: Elyra example pipeline
     mime_type: text/html
     url: https://github.com/elyra-ai/examples/tree/master/pipelines/dax_noaa_weather_data
+  - name: OpenAIHub notebook
+    description: OpenAIHub exmple notebook with data mount
+    mime_type: application/x-ipynb+json
+    url: https://github.ibm.com/OpenAIHub/openaihub/blob/master/notebooks/dlf-notebooks/JFK_Data.ipynb
+    application: OpenAIHub
+    asset_id: notebooks/jfk-airport-analysis
```

Comment:
Perhaps this should be a child property of application since it is only a valid id in the context of whatever is listed as  `application`?

Reply:
yeah, that would make sense, let me do that

Reply:
@ptitzler -- would we do it like this?

```Yaml
  - name: OpenAIHub notebook
    description: OpenAIHub example notebook with data mount
    mime_type: application/x-ipynb+json
    url: https://github.ibm.com/OpenAIHub/openaihub/blob/master/notebooks/dlf-notebooks/JFK_Data.ipynb
    application:
      name: OpenAIHub
      asset_id: notebooks/jfk-airport-analysis
```

Reply:
Done